scalahost/nsc/src/main/scala/scala/meta/internal/scalahost/ScalahostPipeline.scala

```diff
@@ -42,11 +43,17 @@ trait ScalahostPipeline extends DatabaseOps { self: ScalahostPlugin =>
           mminidb.save(scalametaTargetroot, config.sourceroot)
         } catch {
           case NonFatal(ex) =>
-            val msg = new StringWriter()
+            val writer = new StringWriter()
             val path = unit.source.file.path
-            msg.write(s"failed to generate semanticdb for $path:$EOL")
-            ex.printStackTrace(new PrintWriter(msg))
-            global.reporter.error(g.NoPosition, msg.toString)
+            writer.write(s"failed to generate semanticdb for $path:$EOL")
+            ex.printStackTrace(new PrintWriter(writer))
+            val msg = writer.toString
+            config.failures match {
+              case Severity.ERROR => global.reporter.error(g.NoPosition, msg)
+              case Severity.WARNING => global.reporter.warning(g.NoPosition, msg)
+              case Severity.INFO => global.reporter.info(g.NoPosition, msg, force = true)
```

Comment:
No idea, you have to choose however. I suspect `force = false` may mean that the message is not flushed immediately?